runtime/near-vm-runner/src/logic/logic.rs

```diff
@@ -3174,8 +3189,29 @@ bls12381_p2_decompress_base + bls12381_p2_decompress_element * num_elements`
             .checked_sub(&nodes_before)
             .ok_or(InconsistentStateError::IntegerOverflow)?;
         self.result_state.gas_counter.add_trie_fees(&nodes_delta)?;
-        let read =
-            Self::deref_value(&mut self.result_state.gas_counter, storage_read_value_byte, read?)?;
+        let read = match read? {
+            Some(read) => {
+                let read_len = read.len() as u64;
+                // TODO BEFORE UNDRAFTING: REPLACE WITH THE RIGHT CONSTANT USAGE
+                if read_len < 4096 {
```

Comment:
@pugachAG Do you know where the 4k constant I've been told is the limit to load for memtrie lives? I've tried digging around `core/store/src/trie/mem` but haven't been able to find the place in loading the trie that would limit to 4k, and thus to reuse the constant from there.

Reply:
as far as I know inlined values are taken from the flat storage when loading the memtrie, so the constant is `INLINE_DISK_VALUE_THRESHOLD`